backend/app/Http/Controllers/AdminController.php

```
@@ -313,4 +356,280 @@ private function getSubscriptionPrice(string $subscriptionId, int $displaysCount
             return null;
         }
     }
+
+    /**
+     * Show user details page
+     */
+    public function showUser(User $user)
+    {
+        $this->checkAdminAccess();
+        
+        $admin = Auth::user();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$admin'. (undefined)

(UnusedLocalVariable)